reloc/build_reloc.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+. /etc/os-release
+
+print_usage() {
+    echo "build_reloc.sh --clean --nodeps"
+    echo "  --clean clean build directory"
+    echo "  --nodeps    skip installing dependencies"
+    echo "  --version V  product-version-release string (overriding SCYLLA-VERSION-GEN)"
+    exit 1
+}
+
+CLEAN=
+NODEPS=
+VERSION_OVERRIDE=
+while [ $# -gt 0 ]; do
+    case "$1" in
+        "--clean")
+            CLEAN=yes
+            shift 1
+            ;;
+        "--nodeps")
+            NODEPS=yes
+            shift 1
+            ;;
+        "--version")
+            VERSION_OVERRIDE="$2"
+            shift 2
+            ;;
+            *)
+            print_usage
+            ;;
+    esac
+done
+
+VERSION=$(./SCYLLA-VERSION-GEN ${VERSION_OVERRIDE:+ --version "$VERSION_OVERRIDE"})
+# the former command should generate build/SCYLLA-PRODUCT-FILE and some other version
+# related files
+PRODUCT=`cat build/SCYLLA-PRODUCT-FILE`
+DEST="build/$PRODUCT-cqlsh-$VERSION.noarch.tar.gz"
+
+is_redhat_variant() {
+    [ -f /etc/redhat-release ]
+}
+is_debian_variant() {
+    [ -f /etc/debian_version ]
+}
+
+
+if [ ! -e reloc/build_reloc.sh ]; then
+    echo "run build_reloc.sh in top of scylla dir"
+    exit 1
+fi
+
+if [ "$CLEAN" = "yes" ]; then
+    rm -rf build target
+fi
+
+if [ -f "$DEST" ]; then
+    rm "$DEST"
+fi
+
+if [ -z "$NODEPS" ]; then
+    sudo ./install-dependencies.sh
```

Comment:
install-dependencies.sh does not found.
We need following patch to fix:
https://github.com/syuu1228/scylla-cqlsh/commit/bf94cd92ecd17193c40e0294f00432596ac5ca06

Reply:
took this patch too